Assets/Mirage/Runtime/Transport/UdpSocket/UdpTransport.cs

```diff
@@ -0,0 +1,84 @@
+using System.Net;
+using System.Net.Sockets;
+using Mirage.SocketLayer;
+using UnityEngine;
+
+namespace Mirage
+{
+    public sealed class UdpTransport : SocketCreator
```

Comment:
Should probably be called something like `UdpCreator`, since it's not a transport. Also, the Creator suffix sounds kinda weird, but I'm not sure what to replace it with.